ConsoleAppfForGit/Pizza/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Pizza
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // ketchup
+            // sausage
+
+            var pizza = new Pizza();
+            pizza.Ingredients.Add("ketchup");
+            pizza.Ingredients.Add("sausage");
```

Comment:
Now we are adding the components right in here, but it would be cool to pass them in dynamically via `args` + console like `ketchup salami tomato` etc.